presto-main/src/main/java/com/facebook/presto/server/security/SpnegoFilter.java

```diff
@@ -165,7 +165,8 @@ public Principal getUserPrincipal()
                         return;
                     }
                 }
-                catch (GSSException e) {
+                catch (Exception e) {
```

Comment:
GSSException | RuntimeException is indeed a better idea, will update accordingly.